src/components/NewMessageForm/NewMessageForm.vue

```diff
@@ -220,6 +234,10 @@ export default {
 		currentConversationIsJoined() {
 			return this.$store.getters.currentConversationIsJoined
 		},
+
+		hasText() {
+			return this.text !== ''
```

Comment:
When you typed something and remove it with backspace, this condition doesn't turn true anymore 🤔

Reply:
Not the case anymore apparently